OrbitCore/ConnectionManager.cpp

```diff
@@ -73,6 +75,33 @@ void ConnectionManager::InitAsRemote()
     m_Thread = std::make_unique<std::thread>(&ConnectionManager::RemoteThread, this);
 }
 
+//-----------------------------------------------------------------------------
+void ConnectionManager::SetSelectedFunctionsOnRemote( const char* a_Data, size_t a_Size ) {
+    PRINT_FUNC;
+    std::istringstream buffer(std::string(a_Data, a_Size));
+    cereal::JSONInputArchive inputAr( buffer );
+    std::vector<std::string> selectedFunctions;
+    inputAr(selectedFunctions);
+
+    Capture::GSelectedFunctionsMap.clear();
```

Comment:
Could we "unselect" only the selected functions here since we have them at hand (in GSelectedFunctionsMap) instead of iterating through all the functions?

Reply:
I also addressed this change request, but somehow the Github UI does not show it.